read.go

```diff
@@ -456,7 +455,7 @@ func PrintNgramStats(r IndexFile) error {
 		return err
 	}
 	var rNgram [3]rune
-	for ngram, ss := range id.ngrams {
+	for ngram, ss := range id.ngrams.DumpMap() {
```

Comment:
do you wanna consider adding a simpler method here? eg something like `iterate(func(ngram, simpleSection))`.

Reply:
That doesn't feel much simpler-- DumpMap->Iterate would become a couple lines shorter, but the inherent complexity is in traversing the datastructure, not in outputting a map.